packages/instant/src/redux/actions.ts

```diff
@@ -38,6 +39,7 @@ export const actions = {
     updatebuyOrderState: (buyState: AsyncProcessState) =>
         createAction(ActionTypes.UPDATE_SELECTED_ASSET_BUY_STATE, buyState),
     updateLatestBuyQuote: (buyQuote?: BuyQuote) => createAction(ActionTypes.UPDATE_LATEST_BUY_QUOTE, buyQuote),
+    updateSelectedAsset: (assetData?: string) => createAction(ActionTypes.UPDATE_SELECTED_ASSET, assetData),
```

Comment:
It is not used. I think I was just prototyping to see if the structure of things made sense to support this feature. Generally agree that there shouldn't be dead code in a PR. I can remove it, but I do think this will be it's implementation... so probably fine to leave?

Reply:
cool w/ leaving this